ptcg_cockatrice.xml

```diff
@@ -354,7 +354,7 @@
 			<pt>4/4</pt>
 			<tablerow>2</tablerow>
 			<text>{3U}: Evolve from Poliwhirl
-				When you evolve this Pokémon, it gets 3 +1/+1 counters.
+				When this this Pokémon evolves, put three +1/+1 counters on it.
```

Comment:
"this this"

Reply:
Oops. I changed this after seeing your comment but the change isn't reflected here. Do I need to make a new pull request or...?